src/providers/ipfs.js

```diff
@@ -1,11 +1,12 @@
-const ipfsAPI = require('ipfs-api')
+globalThis = global
+const createClient = require('ipfs-http-client')
 const httpProvider = require('./http')()
 
 module.exports = (opts = {}) => {
   // Backwards compatible API: If rpc is passed in options that is passed to IPFS,
   // otherwise all options are provided
   const initIPFS = (ipfsOptions) =>
-    ipfsAPI(ipfsOptions.rpc ? ipfsOptions.rpc : ipfsOptions)
+    ipfsOptions.rpc ? createClient(ipfsOptions.rpc) : createClient()

```

Comment:
I think it should be:
```suggestion
    ipfsOptions.rpc ? createClient(ipfsOptions.rpc) : createClient(ipfsOptions)
```

Reply:
Ok. I included this change and tested with node 10 and 12.